src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,53 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.12.0
+
+### New features
+
+* Added a new `subctl uninstall` command that removes all Submariner components and dataplane artifacts, such as iptables rules and routing
+table entries, from a cluster.
+* Added a new `subctl unexport` command that stops exporting a previously exported service.
+* Added new `subctl cloud prepare` and `subctl cloud cleanup` commands for the Red Hat OpenStack Platform (RHOS).
+* New Globalnet metrics were added:
+  * Count of global Egress IPs allocated at Cluster scope, namespace scope, and for selected pods per CIDR.
+  * Count of global Ingress IPs allocated for Pods/Services per CIDR.
+* Added support for Globalnet objects verification using subctl diagnose command.
+* Added support for `--broker-namespace` flag while deploying the Broker.
+* Added support for running `subctl diagnose` on single node clusters.
+* Added support for running `subctl diagnose` from a pod in cluster.
+* `subctl cloud prepare` now deploys a dedicated gateway node as a default option on GCP and OpenStack platforms.
+* `subctl show` now shows information about the Broker CR in the cluster.
+* `subctl gather` now collects Globalnet information.
+* `subctl diagnose` displays a warning when a generic CNI network plugin is detected.
+* New Service Discovery metric was added:
+  * Count of DNS queries handled by Lighthouse plugin.
```

Comment:
Can we move this to be by the parallel "New Globalnet metrics were added:" sublist? Otherwise, it doesn't make much sense to break this out as a sublist when it only has one item.

Reply:
Doesn't it seem better to group the "metrics added" points? Why keep them so far apart in the list?